CHANGES.md

```diff
@@ -4,6 +4,17 @@
 
 #### @cesium/engine
 
+##### Breaking Changes :mega:
+
+- By default, the screen space camera controller will no longer go inside or under instaces of `Cesium3DTileset`. [#11581](https://github.com/CesiumGS/cesium/pull/11581)
+  - This behavior can be disabled by setting `Cesium3DTileset.disableCameraCollision` to true.
```

Comment:
The option is currently `disableCollision`.
`disableCameraCollision` might be a better name. Or even `disableCameraCollisionCheck` ?

Reply:
While I agree `disableCameraCollision` (I used that first before changing it, which is why this is leftover here) is a clearer name for this particular function, I used `disableCollision` as this is also loads the appropriate data needed for clamping entities to 3D Tiles in https://github.com/CesiumGS/cesium/pull/11604.